ios/Say Their Names/Say Their Names/Controller/Home/Person/PersonIDetailsController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PersonInfoViewController.swift
+//  Say Their Names
+//
+//  Created by Manase on 31/05/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class PersonDetailsController: UIViewController {
+
+    @IBOutlet weak var mediaCollectionView: UICollectionView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mediaCollectionView.delegate = self
+        mediaCollectionView.dataSource = self
+        
+        let personPhotoCell = UINib(nibName: "PersonPhotoCell", bundle: nil)
+        mediaCollectionView.register(personPhotoCell, forCellWithReuseIdentifier: "PersonPhotoCell")
+    }
+}
+
+extension PersonDetailsController: UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 5
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PersonPhotoCell", for: indexPath) as! PersonPhotoCell
```

Comment:
Not a huge fan of force unwrapping, even tho I know this SHOULDN'T cause a crash. I'd personally prefer something along the lines of:
```
let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PersonPhotoCell", for: indexPath)
let personPhotoCell = cell as? PersonPhotoCell
// do something with personPhotoCell
```
What do you think?

Reply:
Yeah, resolving this issue